### PR TITLE
fix: tonal-audibility single-line correction, verifier coverage and environmental corrections

### PR DESCRIPTION
**Pull request by jmrplens**
Standards-correctness fixes for the verifier machinery, environmental acoustics and the GUM uncertainty chain, closing the final domain review; every formula was re-verified against the source texts and the changes below are what deviated.

## Fixed

- **DIN 45681 / ISO PAS 20065 single-line tones.** The Hanning bandwidth correction (1.76 dB) was applied to every tone including single spectral lines, where Formula (7) and the DIN reference program apply none: weak near-bin-centred tones were systematically under-penalized, flipping K_T ratings at the borderline. The correction now applies only to multi-line runs, and the Clause 5.3.8 Step 3 same-band combination runs inside the automated spectrum analysis (with overlapping candidate bands merged into one cluster per chain of tones).
- **Weighting verifier (IEC 61672-1).** The filter response is evaluated at the exact base-10 frequencies the Table 3 design goals are defined at (the nominal-frequency pairing biased deviations by up to 0.27 dB), a dense sweep applies the 5.5.7 larger-of-adjacent limits between the nominal frequencies (a deep notch between nominals previously passed as class 1), and verification at sample rates that cannot cover rows with finite lower limits returns a range-limited verdict instead of full conformance.
- **Wind-turbine tonality (IEC 61400-11).** The 10 dB tone-line rule and the audibility frequency anchor to the highest classified tone line as 9.5.3/9.5.4 require (not the probed candidate), the 9.5.2 possible-tone screen is applied, and spectra without an identified tone now say so instead of fabricating one (they must be excluded from the 9.5.1 bin averaging, which is documented).
- **ISO 1996-2 measurement chain.** The repeated-measurement uncertainty implements the primary Formulae (17)+(19) energy-domain route (the previous code was the Note 2 approximation, mislabelled, overstating the uncertainty by up to 8 dB on spread levels, and is kept as a documented approximation with a spread warning); the 3 dB residual-correction wording is corrected (the uncorrected level is the reportable upper bound) and percentile ordering is validated.
- **ISO 9613-2 barrier.** A negative path difference (line of sight clearing the barrier top) is now representable and evaluated per the text after Eq. (16) with Kmet = 1, removing up to 20 dB of spurious screening and the discontinuity at grazing; the atmospheric-absorption term uses exact midband frequencies as the standard's own Table 2 does.
- **GUM machinery.** Welch-Satterthwaite is not applied to correlated budgets (the fallback returns infinite effective degrees of freedom with a warning; the previous path could return coverage factors of 10^151), and the finite-difference sensitivity step is scaled to the input magnitude so large values with small uncertainties no longer silently lose their contribution entirely.
- Smaller items: the NT ACOU 112 onset-rate gate, tone-frequency application ranges, calibration input validation, Monte Carlo trial validation, nominal-snapping and extrapolation notes, and docstring corrections throughout.

## Added

- **Published oracles**: the DIN 45681 Anhang I wind-energy-plant example (a second end-to-end oracle from the parent standard), GUM Annex H.1 propagated end-to-end and Annex H.2 as the only published correlated-path oracle, Supplement 1 Tables 2-3 (including a seeded Monte Carlo conformance entry), the full ISO 9613-2 Table 2 attenuation grid at exact midbands, and the IEC 61260-1 Table F.1 breakpoints.
- An adversarial detuning battery documents that the filter-class verifier's thresholds are the standard's own: a 0.5 percent centre-frequency detune passes, 1 percent fails, with margins matching an independent dense-grid mask.

Conformance grows from 244 to 251 checks. Full suite green (3097 tests, zero skips), ruff and mypy clean.

<a href="https://cubic.dev/pr/jmrplens/phonometry/pull/174?utm_source=github" target="_blank" rel="noopener noreferrer" data-no-image-dialog="true"><picture><source media="(prefers-color-scheme: dark)" srcset="https://www.cubic.dev/buttons/review-in-cubic-dark.svg"><source media="(prefers-color-scheme: light)" srcset="https://www.cubic.dev/buttons/review-in-cubic-light.svg">Review in cubic</picture></a>

## Summary by CodeRabbit

* **New Features**
  * Added richer tonal/impulse results: tone grouping sizes, qualification masks, and explicit “identified tone” reporting (plus new warning types).
  * Added outdoor/IEC additions: `Barrier` line-of-sight-clear handling, vertical rotor-axis support, and weighting verification now includes a between-nominals sweep control and results.
  * Enhanced uncertainty outputs: `reportable_upper_bound` and `approximate_uncertainty`.
* **Bug Fixes**
  * Improved correctness across standards calculations and edge-case validation/reporting (residual correction, tone grouping, GUM/Monte Carlo semantics, and outdoor propagation geometry).
* **Documentation**
  * Updated API docs and multiple guides; refreshed conformance counts and added errata.
* **Performance**
  * Faster ECAC noise-contour grid evaluation with equivalent results.
* **Tests**
  * Expanded oracle and edge-case coverage across ISO/IEC/GUM/psychoacoustics.

### CODE REVIEW

File: src/phonometry/environmental/measurement.py [hunk 1]
```diff
@@ -482,22 +527,47 @@ def uncertainty_from_repeated_measurements(
...
         raise ValueError("'levels' must be finite.")
     n = int(lev.size)
-    mean_level = 10.0 * np.log10(np.mean(10.0 ** (0.1 * lev)))
-    uk = np.sqrt(np.sum((lev - mean_level) ** 2) / (n - 1))
+    energies = 10.0 ** (0.1 * lev)
+    mean_energy = float(np.mean(energies))
+    mean_level = 10.0 * np.log10(mean_energy)
+    sk = float(np.sqrt(np.sum((energies - mean_energy) ** 2) / (n - 1)))
+    uk = 10.0 * np.log10(mean_energy + sk) - mean_level
+    uk_approx = float(np.sqrt(np.sum((lev - mean_level) ** 2) / (n - 1)))
```
[the remark this answers is not in the document]
Comment (jmrplens) [reply src/phonometry/environmental/measurement.py:558]: Declined after re-reading the print: Formula (20) as published is uk = sqrt(sum (Li - Lk)^2 / (nk - 1)) where Lk is defined by Formula (18), the energy mean of the same clause, not the arithmetic mean. The module implements that letter (12.18 dB for [50, 60, 70]); a std about the arithmetic mean would be a different statistic that only coincides inside Note 2's own validity condition (small level differences), where the two means converge. The primary Formulae (17)+(19) route is the recommended result either way, and the spread warning fires exactly when the approximation degrades.

**Comment by jmrplens** [comment]
All review threads are now addressed. From the analysis: the flagged complexities moved into dedicated helpers and the float-equality guards became direct comparisons. From gemini: the Note 2 approximation stays on the printed Formula (20) (its Lk is the Formula (18) energy mean by definition; the arithmetic-mean variant only coincides inside the note's own validity condition), with the primary Formulae (17)+(19) result recommended either way. From CodeRabbit, nine of ten applied: candidate-tone validation and the documented fallback, the processing-Nyquist breakpoint cutoff, range-limited verdicts for any dropped Table 3 row, a non-finite guard in the ISO 9613-2 conformance check, a locality-preserving finite-difference step with a nonlinear regression, an explicit coverage-factor policy for correlated budgets (the GUM defines no correlated Welch-Satterthwaite form), and the documentation rows with their Spanish mirrors. The adaptive-refinement suggestion for the between-nominals sweep is resolved by documenting the sweep as sampled with a configurable density instead.
